src/site/stages/build/process-cms-exports/validator.js

```diff
@@ -1,20 +1,31 @@
+const fs = require('fs');
+const path = require('path');
 const Ajv = require('ajv');
 
 const ajv = new Ajv();
 
-const {
-  entityReference,
-  entityReferenceArray,
-} = require('./schemas/common/entity-reference');
-const genericNestedString = require('./schemas/common/generic-nested-string');
-
 // Custom formats
 ajv.addFormat('custom-date-time', /\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}/);
 
-// Common $ref schemas
-ajv.addSchema(entityReference, 'EntityReference');
-ajv.addSchema(entityReferenceArray, 'EntityReferenceArray');
-ajv.addSchema(genericNestedString, 'GenericNestedString');
+/**
+ * Loads all the schemas in a directory into ajv
+ */
+const loadSchemas = dir => {
+  fs.readdirSync(dir).forEach(fileName => {
+    // eslint-disable-next-line import/no-dynamic-require
+    const schema = require(path.join(dir, fileName));
+    ajv.addSchema(
+      schema,
+      // Common schemas should have $id, but dynamically name raw and transformed schemas
+      schema.$id ||
+        `${dir.split(path.sep).pop()}/${path.basename(fileName, '.js')}`,
```

Comment:
Alternatively, we can pass in two parameters—one for the path, and one for the namespace. I'm fine either way. `dir.split(path.sep).pop()` is kinda gross, but then again, so is passing in more parameters than we need. :man_shrugging:

Reply:
It isn't _super_ obvious that this is supposed to be `raw/[entityType]` or `transformed/[entityType]`, but I think it's ok for the time being.  It think it's more useful to have this in so we can have the validation on our upcoming transformer PRs